contracts/oracles/OpenOracleFrameworkOracle.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+// Using the same Copyleft License as in the original Repository
+pragma solidity 0.6.12;
+import "@boringcrypto/boring-solidity/contracts/libraries/BoringMath.sol";
+import "../interfaces/IOracle.sol";
+
+// Open Oracle Framework Interface
+interface IOpenOracleFramework {
+    function getFeed(uint256 feedID) external view returns (uint256, uint256, uint256);
+}
+
+// Open Oracle Framework TWAP Interface
+interface IOpenOracleFrameworkTWAP {
+    function getTWAP(
+        address OOFContract,
+        uint256[] memory feedIDs,
+        uint256[] memory timestampstart,
+        uint256[] memory timestampfinish,
+        bool strictMode)
+    external view returns (uint256[] memory TWAP);
+}
+
+contract OpenOracleFramework is IOracle {
+    using BoringMath for uint256; // Keep everything in uint256
+
+    // Calculates the latest rate
+    function _get(
+        address oracle,
+        uint256 feedId
+    ) internal view returns (uint256) {
+        uint256 price;
+        uint256 decimals;
+        (price, , decimals) = IOpenOracleFramework(oracle).getFeed(feedId);
+        return price / decimals;
```

Comment:
Does this already conform to the Kashi oracle standard of providing the price of 1e18 of asset?

Reply:
We standardize all feed returns to be 18 dec yes.